Dockerfile

```diff
@@ -0,0 +1,52 @@
+# hadolint global ignore=DL3008
+FROM debian:buster
+
+RUN apt-get update && apt-get install -y --no-install-recommends \
+      python \
+      python-pip \
+      python-dev \
+      python-setuptools \
+      ca-certificates \
+      gcc \
+      libpq-dev \
+      curl \
+      gunicorn \
+    && rm -rf /var/lib/apt/lists/*
+
+SHELL ["/bin/bash", "-o", "pipefail", "-c"]
+
+# kelseyhightower confd arm64 builds are broken: https://github.com/kelseyhightower/confd/issues/850
+ARG CONFD_VERSION=0.19.1
+RUN CONFD_ARCH=$(arch | sed s/aarch64/arm64/ | sed s/x86_64/amd64/) \
+    && curl -SL "https://github.com/abtreece/confd/releases/download/v${CONFD_VERSION}/confd-v${CONFD_VERSION}-linux-${CONFD_ARCH}.tar.gz" | tar -xz -C /usr/local/bin/ \
+    && /usr/local/bin/confd -version
```

Comment:
Is there seriously nothing packaged we can use for this?

Reply:
I've previously used `confd` because of it's flexibility and I quite like it. Unfortunately`confd` isn't packaged in Ubuntu or Debian. The install is complicated by adding in the ARM64 support.

Using `envsubst` from gettext could likely work as a substitute. Alternatively I could likely just use python's ability to read directly environment variables.

Reply:
Replaced with simplified `gettext envsubst` using distro package. 👍